xbmc/FileItem.cpp

```diff
@@ -2021,7 +2021,7 @@ void CFileItemList::Sort(SortDescription sortDescription)
   }
 
   // replace the current list with the re-ordered one
-  m_items.assign(sortedFileItems.begin(), sortedFileItems.end());
+  m_items = std::move(sortedFileItems);
```

Comment:
CFileItemList is commonly used, but this is code inside CFileItemList! CFileItemList cannot possibly have an attribute of type CFileItemList, can it?